assignments/objects.js

```diff
@@ -17,29 +17,66 @@ let example = {
   "gender": "F"
 }
 
+
+
+
 // Write your intern objects here:
 
+function InternInfo(id, name, email, gender) {
+  this.id = id;
+  this.name = name;
+  this.email = email;
+  this.gender = gender;
+}
+
```

Comment:
Nice job using the this keyword! You are ahead of the game sir. Very nicely done!